client/post-editor/post-editor.jsx

```diff
@@ -849,7 +843,7 @@ const PostEditor = React.createClass( {
 
 		// make sure the history entry has the post ID in it, but don't dispatch
 		page.replace(
-			basePath + '/' + this.props.sites.getSite( post.site_ID ).slug + '/' + post.ID,
+			this.props.editPath,
```

Comment:
On an unrelated note, you've shortened this sufficiently that it'd probably be fine on a single line.